src/xoak/tests/conftest.py

```diff
@@ -64,6 +72,34 @@ def query_brute_force(dataset, dataset_dims_shape, indexer, indexer_dims_shape,
     return dataset.isel(indexers=pos_indexers)
 
 
+def query_brute_force_indexers(
```

Comment:
We could avoid duplicating this function and have `query_brute_force` simply return `pos_indexers`.